### DISCUSSION TITLE
Multi-send in Windows version - cant get it to work, what am I missing?

### DISCUSSION THREAD

**Discussion by lrc-cc** [dannagle/PacketSender#247] [Q&A]
I have PacketSender v 7.2.3 installed in Windows 10.  I cannot figure out how to get multi-send to work,  hope somebody can help.

I can select multiple packets in the saved packets list like this:

![image](https://user-images.githubusercontent.com/30603650/127715932-040798cd-d22a-4121-b3d2-9efa48069ea9.png)

... but then I can find no way to actually send them.  No "multi-send" button appears,  right clicking the selected packets does not give a multi-send option, and if I just click one of the usual "Send" buttons for any of the selected packets then just that packet is sent.

What am I missing?  Is there some non-obvious keyboard/mouse combination required to multi-send?  Or ???

Thanks very much for your help!

LRC-CC

**Comment by dannagle** [comment]
Hello, multi-send was removed in favor of a newer method that is currently in the development branch. I was hoping to have it released by now, but that schedule has been delayed.

There is no full release yet with the replacement, but you can join alpha testing if you email hello AT packetsender.com

**Reply by lrc-cc** [reply]
Hi Dan,

Thanks for the info!  I'll consider the alpha testing but in the meantime would it be possible to get an older release version that has multi-send?  I have an immediate need for this to help debug some firmware on a product I'm developing.

Thanks,
LRC-CC

**Reply by dannagle** [reply]
Yes, old releases are archived here:
https://github.com/dannagle/PacketSender/releases

Multi-send was removed in v7.  You will need to jump down to a v6 release.
